e2e/wdio/core/tests/tile.e2e-spec.ts

```diff
@@ -0,0 +1,114 @@
+
+describe('Textarea component test', function () {
+    const tilePage = new TilePo();
+    const {
+        tile,
+        fdTile,
+        launchExample,
+        kpiExample, 
+        feedExample,
+        numericContent,
+        actionExample,
+        closeButton, 
+        moreButton,
+        bagdeExample,
+        badge,
+        tileContent,
+        tileFooter,
+        tileHeader,
+        columnsExample
+    } = tilePage;
+
+    beforeAll(() => {
+        tilePage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForPresent(tilePage.title);
+    }, 1);
+
+    it('should check that tile clickable', () => {
+        for (let i = 0; i < tile.length; i++) {
+            expect(isElementClickable(tile, i)).toBe(true, `tile with index ${i} is not clickable`);
+        }
+    });
+
+    it('should check size in default example', () => {
+        expect(getAttributeByName(launchExample + fdTile, 'size', 1)).toBe('s');
+        expect(getAttributeByName(launchExample + fdTile, 'size', 3)).toBe('s');
+    })
+
+    it('should check launch example', () => {
+        checkTileType(launchExample, 'launch');
+    });
+
+    it('should check kpi example', () => {
+        checkTileType(kpiExample, 'kpi');
+    });
+
+    it('should check feed example', () => {
+        checkTileType(feedExample, 'feed');
+    });
+
+    it('should check size of kpi example', () => {
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 2)).toBe('m');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 3)).toBe('m');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 4)).toBe('s');
+        expect(getAttributeByName(kpiExample + numericContent, 'size', 5)).toBe('s');
+    });
+
+    it('should check action example', () => {
+        expect(getElementClass(actionExample + tile)).toContain('action');
+        expect(isElementClickable(closeButton)).toBe(true, 'close button is not clickable');
+        expect(isElementClickable(moreButton)).toBe(true, 'more button is not clickable');
+    });
+
+    it('should check badge in badge example', () => {
+        expect(isElementDisplayed(bagdeExample + tile + badge)).toBe(true, 'badge lable is not displayed');
+    });
+
+    it('should check two columns in columns example', () => {
+        expect(getElementClass(columnsExample + tileHeader)).toContain('2-col');
+        expect(getElementClass(columnsExample + tileContent)).toContain('2-col');
+        expect(getElementClass(columnsExample + tileFooter)).toContain('2-col');
+    });
+
+    it('should check visual regression for all examples', () => {
+        tilePage.saveExampleBaselineScreenshot();
+        expect(tilePage.compareWithBaseline()).toBeLessThan(5);
+    });
+
+    it('should check RTL and LTR orientation', () => {
+        tilePage.checkRtlSwitch();
+    });
+
+    function checkTileType(section: string, type: string) : void{
```

Comment:
formatting issue
should be function checkTileType(section: string, type: string): void {

Reply:
@maxym-valor still missing space between `void` and `{`  
`...string): void {`